contrib/win32/win32compat/termio.c

```diff
@@ -42,7 +42,7 @@
 
 #define TERM_IO_BUF_SIZE 2048
 
-extern int in_raw_mode;
+extern BOOL isAnsiParsingRequired;
```

Comment:
this looks inaccurate. in_raw_mode is modified at run time as part of calls to sshpty* routines.